src/Elastic.Extensions.Logging.Console/EcsConsoleFormatterOptions.cs

```diff
@@ -0,0 +1,28 @@
+using Elastic.CommonSchema;
+using Elastic.Extensions.Logging.Common;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+
+namespace Elastic.Extensions.Logging.Console;
+
+/// <summary>  </summary>
+public class EcsConsoleFormatterOptions : ConsoleFormatterOptions, ILogEventCreationOptions
+{
+	/// <inheritdoc cref="IEcsDocumentCreationOptions.IncludeHost"/>
+	public bool IncludeHost { get; set; } = true;
+
+	/// <inheritdoc cref="IEcsDocumentCreationOptions.IncludeProcess"/>
+	public bool IncludeProcess { get; set; } = true;
+
+	/// <inheritdoc cref="IEcsDocumentCreationOptions.IncludeUser"/>
+	public bool IncludeUser { get; set; } = true;
+
+	/// <inheritdoc cref="IEcsDocumentCreationOptions.IncludeActivityData"/>
+	public bool IncludeActivityData { get; set; } = true;
+
+	/// <inheritdoc cref="ILogEventCreationOptions.Tags"/>
+	public string[]? Tags { get; set; }
```

Comment:
I'm becoming a fan of avoiding nulls like this. Would making this default to an empty array and making it non-nullable be an option?

Reply:
It would but this would be breaking since we inherit this from `ILogEventCreationOptions` and `ElasticsearchLoggerOptions` who also implements it is already released.

Reply:
Gotcha! Fair enough.